guardrails/cli/configure.py

```diff
@@ -43,15 +37,13 @@ def configure(
     try:
         notice_message = """
 
-    You can find your tokens at https://hub.guardrailsai.com/tokens
+    You can find your token at https://hub.guardrailsai.com/tokens
```

Comment:
Since this url is static, we could just open it for them.  Thoughts?

Reply:
Yes let's have it as an issue